extensions/amp-ad-network-adsense-impl/0.1/test/test-amp-ad-network-adsense-impl.js

```diff
@@ -23,12 +23,16 @@ describe('amp-ad-network-adsense-impl', () => {
   let sandbox;
   let adsenseImpl;
   let adsenseImplElem;
+  let getSigningServiceNamesMock;
 
   beforeEach(() => {
     sandbox = sinon.sandbox.create();
     adsenseImplElem = document.createElement('amp-ad');
     adsenseImplElem.setAttribute('data-ad-client', 'adsense');
     adsenseImpl = new AmpAdNetworkAdsenseImpl(adsenseImplElem);
+    getSigningServiceNamesMock = sandbox.stub(
```

Comment:
do we plan to use the stub in other places? if not, it can be simplified to
```
sandbox.stub(AmpAdNetworkAdsenseImpl.prototype, 'getSigningServiceNames', () => {
  return ['google'];
})
```